apps/mobile/src/components/InsideRouter/components/SettingsScreen/atoms.ts

```diff
@@ -1,4 +1,23 @@
 import {atom} from 'jotai'
+import {sessionDataOrDummyAtom} from '../../../../state/session'
 
 export const reportIssueDialogVisibleAtom = atom<boolean>(false)
 export const changeCurrencyDialogVisibleAtom = atom<boolean>(false)
+export const qrCodeDialogVisibleAtom = atom<boolean>(false)
+
+export const encodedUserDetailsUriAtom = atom<string>((get) => {
+  const session = get(sessionDataOrDummyAtom)
+
+  const userDetails = {
+    name: session.realUserData.userName,
+    label: 'Scanned from qr code',
+    numberToDisplay: session.phoneNumber,
+    imageUri: session.realUserData.image.imageUri,
```

Comment:
Remove imageURI. We can not display it on other user phone since the URI is path to a file